kernelci/rootfs.py

```diff
@@ -1,5 +1,6 @@
-# Copyright (C) 2019 Collabora Limited
+# Copyright (C) 2019, 2021 Collabora Limited
```

Comment:
```suggestion
# Copyright (C) 2019, 2020, 2021 Collabora Limited
```
2020 should have been added, there were several commits last year.

Reply:
added